rotkehlchen/db/evmtx.py

```diff
@@ -686,20 +708,28 @@ def get_or_create_genesis_transaction(
                 )
         return tx
 
-    def _form_evm_transaction_dbquery(self, query: str, bindings: list[Any], has_premium: bool) -> tuple[str, list[tuple]]:  # noqa: E501
+    def _form_evm_transaction_dbquery(self, query: str, bindings: list[Any], has_premium: bool) -> tuple[str, list]:  # noqa: E501
```

Comment:
update docstring here with the new details since this no longer concerns only one table